package.json

```diff
@@ -60,13 +60,13 @@
     "sinon": "^15.0.0"
   },
   "engines": {
-    "node": "14.x"
+    "node": "18.x"
   },
   "pre-commit": [
     "lint"
   ],
   "volta": {
-    "node": "14.21.1",
+    "node": "18.15.0",
```

Comment:
Isn't this a major version upgrade, is everything working as expected?
Is this tested thoroughly?

Reply:
Yes, this is a major version upgrade.
As far as testing is considered, I have considered these things:

1. Checking if the backend is successfully set up or not.

![image](https://user-images.githubusercontent.com/57326919/228008319-28b92d6d-49aa-4ec6-abd0-f542f372a848.png)

2. Running unit tests.

![image](https://user-images.githubusercontent.com/57326919/228008503-d69ae0b4-67ca-4413-a790-a883ee711c0e.png)

3. Running integration tests.

![image](https://user-images.githubusercontent.com/57326919/228008630-be96e43f-b8d6-4eb6-bf7a-e9c71683d8cb.png)

Are these the tests it needs to pass or is there anything else that is required to be considered? 
Reference: 
https://github.com/Real-Dev-Squad/website-backend/blob/fix-node-version-update/CONTRIBUTING.md#pull-request-guidelines